### ISSUE TITLE
No release notes in tag

### ISSUE THREAD

**Issue by jscancella** [semantic-release/semantic-release#2338]
## Current behavior

When running in gitlab, a new tag is created, but the generated release notes are not included in the tag's release notes

## Expected behavior

The release notes generated during the semantic-release are included in gitlab tag's release notes

## Environment

- **semantic-release** version: 18.0.0
- CI environment: gitlab
- Plugins used: commit-analyzer, release-notes-generator
- **semantic-release** configuration: N/A
- CI logs: N/A

**Comment by bbartley** [comment]
Can confirm, we are seeing this in Gitlab as well on v19.0.2.  We are using semantic-release in the context of a large distributed systems architecture for a government program.  Even though we aren't totally blocked by this, it is a bit of a stumbling block, because it impacts transparency into our system components.

**Comment by bbartley** [comment]
Thanks @jscancella!   When you say "install the needed tools", does that mean you had to explicitly install the plug-ins, e.g. `release-notes-generator`?  I was expecting it to be installed by default--I thought I saw that in the documentation somewhere.

**Comment by jscancella** [comment]
Yes, looking at my test project, I had to explicitly do a `npm install` before calling the `npx semantic-release` and then create a package.json file with the needed info.
